docs/quickstarts/nextjs.mdx

```diff
@@ -34,205 +35,172 @@ To get started using Clerk with Next.js, add the SDK to your project:
 
 Select your preferred router below to learn how to make this data available across your entire app:
 
-<CodeBlockTabs type="router" options={["App Router", "Pages Router"]}>
-    ```tsx filename="/src/app/layout.tsx" {1,10-14}
-    import { ClerkProvider } from '@clerk/nextjs'
-    import './globals.css'
-
-    export default function RootLayout({
-      children,
-    }: {
-      children: React.ReactNode
-    }) {
+<Tabs type="router" items={["App Router", "Pages Router"]}>
+  <Tab>
+  <Callout type="info">
+    If you are not using `src`, the `app/layout.tsx` file will be in your root folder.
+  </Callout>
+
+  ```tsx filename="src/app/layout.tsx" {1,10-14}
+  import { ClerkProvider } from '@clerk/nextjs'
+  import './globals.css'
+
+  export default function RootLayout({
+    children,
+  }: {
+    children: React.ReactNode
+  }) {
+    return (
+      <ClerkProvider>
+        <html lang="en">
+          <body>{children}</body>
+        </html>
+      </ClerkProvider>
+    )
+  }
+  ```
+  </Tab>
+
+  <Tab>
+  <Callout type="info">
+    If you are not using `src`, the `_app.tsx` file will be in your root folder.
+  </Callout>
+
+  ```tsx filename="/src/pages/_app.tsx" {2,6-8}
+    import '@/styles/globals.css'
+    import { ClerkProvider } from "@clerk/nextjs/server";
+    import type { AppProps } from "next/app";
+    function MyApp({ Component, pageProps }: AppProps) {
       return (
-        <ClerkProvider>
-          <html lang="en">
-            <body>{children}</body>
-          </html>
+        <ClerkProvider {...pageProps}>
+          <Component {...pageProps} />
         </ClerkProvider>
       );
     }
     export default MyApp;
-    ```
-
-    ```tsx filename="_app.tsx" {2,6-8}
-      import '@/styles/globals.css'
-      import { ClerkProvider } from "@clerk/nextjs/server";
-      import type { AppProps } from "next/app";
-      function MyApp({ Component, pageProps }: AppProps) {
-        return (
-          <ClerkProvider {...pageProps}>
-            <Component {...pageProps} />
-          </ClerkProvider>
-        );
-      }
-      export default MyApp;
-      ```
-
-</CodeBlockTabs>
+  ```
+  </Tab>
+</Tabs>
 
-### Add authentication to your app with middleware
+### Add Middleware to your application
 
-Now that Clerk is installed and mounted in your application, you can decide which pages are public and which should require authentication to access:
+Middleware grants you access to user authentication state throughout your application, on any route or page. With `clerkMiddleware()` offers the capability to protect routes, it does not enforce protection by default. Please see the [`clerkMiddleware()` reference](/docs/references/nextjs/clerk-middleware) and the [Route Protection guide](/docs/references/nextjs/route-protection) for information on protecting routes.

```

Comment:
How do we feel about adding this little sentence - so it's super clear:

```suggestion
Middleware grants you access to user authentication state throughout your application, on any route or page. While `clerkMiddleware()` offers the capability to protect routes, it does not enforce protection by default. All routes will be public and you must opt-in to protection. Please see the [`clerkMiddleware()` reference](/docs/references/nextjs/clerk-middleware) and the [Route Protection guide](/docs/references/nextjs/route-protection) for information on protecting routes.

```

or another way that I wrote it is:

By default, `clerkMiddleware()` will not protect any routes. All routes are public and you must opt-in to protection for routes. See the [`clerkMiddleware()` docs](https://github.com/docs/references/nextjs/clerk-middleware) to learn how to require authentication for specific routes. 

---

I just want it to be very clear to users that *all routes are public by default*. Saying routes are "protected" may not be clear to users that "protection" = "requires authentication / requires sign-in"